### ISSUE TITLE
Show more details when an object has been selected

### ISSUE THREAD

**Issue by jalil1408** [Stellarium/stellarium#487]
* Lower/Upper **Transit** (Meridian) : Time, Az, Alt, Disk Duration
* Lower/Upper **Culmination** (Max/Min Alt) : Time, Az, Alt, Disk Duration
* **Mid Day/Night** (Half Set-Rise) : Time, Az, Alt
* **Rise/Set** (at alt=0 and at visible alt) : Time, Az, Alt, Disk Duration
* **Shadow** factor

**Comment by lrm25** [comment]
Is this issue still available and do you still want it done?  I could work on it if so.

**Comment by gzotti** [comment]
Hi! Alex has added rise/set/transit, but I think no shadow factor or disk duration. Rise /Set/Transit may have some slight issues (#554), should be reviewed. So yes, please, go ahead!

**Comment by lrm25** [comment]
OK, I can take it.  One big problem though:  I am not completely certain what shadow factor or disk duration are, and couldn't find info online.  Could you please let me know?  Thanks.

**Comment by gzotti** [comment]
Shadow factor sounds like eclipse details. Usually for Lunar eclipses: percentage of disk diameter in penumbra and umbra, if possible maximum percentage; for Solar, it has been done already.
Disk duration: could be time it takes to transit the meridian (just diameter in RA? i.e. diameter/cos(declination)) or for Mercury/Venus to transit Solar disk. (Adapt algorithm for possible transits seen from the outer planets?) But @jalil1408 should give an explanation.

**Comment by jalil1408** [comment]
shadowFactor = 1 / tan(sunAltitude)

disk transit duration = time from [the west limb at transit] to [the east limb at transit]

**Comment by alex-w** [comment]
Seems shadow factor applicable for Sun only (and without relation to the eclipses) and disk transit duration applicable for inner solar system bodies (and moons on observer's planet).

**Comment by gzotti** [comment]
@jalil1408 in astronomy "transit" may mean "transit of any object through the meridian" or "transit through the Solar disk" (or even "Transit of a Iovian satellite through Jupiter's disk" or such). Please specify this clearly. It seems you are thinking of transit of the Solar disk (only) through the meridian (taking about 2 minutes), or duration from "first flash" to "solar disk completely visible" on the horizon.  
Mid day/night and (solar) transit (of the meridian) are the same things. Az=0° or 180°. 
And please give an example for the Guide why shadowFactor=cotan(sunAltitude) is important for the average amateur astronomer user. 
I know all these data are relevant for some religious purposes, therefore all this should be moved to a dedicated plugin then. (With Hilal sighting estimate, prayer times etc. While we could probably do that, it is certainly not to be seen as authoritative answer unless implemented by the official relevant authorities!).

**Comment by alex-w** [comment]
@jalil1408 is shadow factor an equal to shadow length, when object has 1 meter of height?

**Comment by gzotti** [comment]
@alex-w yes, this factor is used for determination of Islamic afternoon prayer time. (Not necessarily 1 meter, but "multiples of vertical height of a pole put onto the ground") Also Noon prayer time has to do with his transit duration. These are cultural topics, should go into a dedicated plugin.

**Comment by alex-w** [comment]
Looks like this is good task for specific plugin

**Comment by alex-w** [comment]
This is a good task for the community to participate in the contribution into Stellarium. Who wants to help us?
